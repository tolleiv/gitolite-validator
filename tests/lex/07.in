# managers should be able to read any repo
repo @all
    R                       =   @managers

# ...other rules for other repos...

repo foo bar
    RW+                     =   alice @teamleads
    -   master              =   dilbert @devteam
    -   refs/tags/v[0-9]    =   dilbert @devteam
    RW+ dev/                =   dilbert @devteam
    RW                      =   dilbert @devteam
    R                       =   @managers
