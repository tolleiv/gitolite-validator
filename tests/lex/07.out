{t_comment[1]  managers should be able to read any repo}
{t_eol[1] }
{t_string[2] repo}
{t_group[2] all}
{t_eol[2] }
{t_string[3] R}
{t_assign[3] }
{t_group[3] managers}
{t_eol[3] }
{t_eol[4] }
{t_comment[5]  ...other rules for other repos...}
{t_eol[5] }
{t_eol[6] }
{t_string[7] repo}
{t_string[7] foo}
{t_string[7] bar}
{t_eol[7] }
{t_string[8] RW+}
{t_assign[8] }
{t_string[8] alice}
{t_group[8] teamleads}
{t_eol[8] }
{t_dash[9] }
{t_string[9] master}
{t_assign[9] }
{t_string[9] dilbert}
{t_group[9] devteam}
{t_eol[9] }
{t_dash[10] }
{t_string[10] refs/tags/v[0-9]}
{t_assign[10] }
{t_string[10] dilbert}
{t_group[10] devteam}
{t_eol[10] }
{t_string[11] RW+}
{t_string[11] dev/}
{t_assign[11] }
{t_string[11] dilbert}
{t_group[11] devteam}
{t_eol[11] }
{t_string[12] RW}
{t_assign[12] }
{t_string[12] dilbert}
{t_group[12] devteam}
{t_eol[12] }
{t_string[13] R}
{t_assign[13] }
{t_group[13] managers}
{t_eol[13] }
